test/python/WMCore_t/WebTools_t/RESTFormat_t.py

```diff
@@ -74,7 +76,9 @@ def testReturnFormat(self):
                          output={'code':200, 'data':'{"a": "a%", "b": "b"}'})
 
         url = self.urlbase + 'list?input_int=a&input_str=a'
-        expected_data = '''{"exception": 400, "message": "Invalid input: Input data failed validation.", "type": "HTTPError"}'''
+        expected_data_py2 = '{"exception": 400, "message": "Invalid input: Input data failed validation.", "type": "HTTPError"}'
+        expected_data_py3 = '{"exception": 400, "type": "HTTPError", "message": "Invalid input: Input data failed validation."}'
+        expected_data = expected_data_py3 if PY3 else expected_data_py2
         methodTest('GET', url, accept=return_type,
                          output={'code':400, 'data':expected_data})
```

Comment:
I know this is not an elegant solution, but having a proper fix would require either a significant change of `methodTest`, ar writing a new `methodTest`. Would it be worth it?